frontend/src/components/borrow/RequestLoan.vue

```diff
@@ -7,61 +7,92 @@
+            <label> INTEREST PERMILS </label>
+            <md-input type="number" v-model="interestPermil" />
+            <div class="rl-input-right-text">	&#8240;</div>
           </md-input-container>
         </div>
       </div>
-      <div class="rl-payback">
-        <div> Payback time period </div>
-        <div class="rl-payback-time"> 7 DAYS </div>
+      <div class="rl-payback-container">
+        <div class="rl-payback">
+        <md-input-container>
+          <label> PAYBACK TIME </label>
+          <md-input v-model="paybackTime" type="number" />
+          <div class="rl-input-right-text"> DAY(S) </div>
+          </md-input-container>
+        </div>
+      </div>
+      <div class="rl-fundraising">
+        <div> Fundraising time period </div>
+        <div class="rl-fundraising-time"> 7 DAYS </div>
       </div>
       <div class="rl-description">
         <md-input-container>
           <label> Description </label>
           <md-textarea v-model="description" maxlength="800" />
         </md-input-container>
       </div>
-      <div class="rl-metamask">
-        <div class="rl-metamask-to-complete"> To complete process you need metamask account </div>
-        <div class="rl-metamask-what-is"> What is metamask and how it works <a href="/"> Read here </a> </div>
-        <div class="rl-metamask-button-container">
-          <purple-button text="AUTHORIZE METAMASK"  @click.native='requestLoan' />
+      <div class="rl-metamask" :class="isLoading()">
+        <purple-button class="rl-request-button" text="REQUEST LOAN"  @click.native='requestLoan' />
+        <div class="rl-loader">
+          <spinner />
+          <div class="rl-metamask-text">
+            Please don't close this window and wait until we process your order. It usually takes about 10 seconds
+          </div>
         </div>
       </div>
     </div>
   </div>
 </template>
 
-<script>
-import PurpleButton from '../common/PurpleButton';
-import API from '../../api';
+<script lang="ts">
+import Vue from 'vue'
 
-export default {
+import PurpleButton from '../common/PurpleButton.vue';
+import Spinner from '../common/Spinner.vue';
+import API, { Loan } from '../../api';
+import { goToLoan } from '../../router';
+
+const Component = Vue.extend({
   name: 'RequestLoan',
   data() {
     return {
-      ethAmount: '0.01',
-      petcentage: '5',
+      amount: '0.01',
+      interestPermil: '5',
       description: '',
+      paybackTime: '7',
+      loading: false,
     };
   },
   components: {
     'purple-button': PurpleButton,
+    'spinner': Spinner,
   },
   methods: {
+    isLoading: function isLoading():string {
+      return this.loading ? 'rl-loading' : '';
+    },
     requestLoan: async function requestLoan() {
-      await API.instance().addNewLoan('ttt', 1, 2, 3, 4);
+      this.loading = true;
+      const daysToSeconds = seconds => seconds * 24 * 60 * 60;
+      const loan: Loan = await API.instance().addNewLoan(
```

Comment:
The API has changed, please pull from master.

It's going to be something like:
````API.instance().newLoan(this.description, this.amount, this.interestPermil, moment().add(7, 'days'), moment().add(whatever, 'days');````